docs/reference/NuGet-Client-SDK.md

```diff
@@ -34,12 +34,33 @@ You can find the source code for these packages in the [NuGet/NuGet.Client](http
 
 ## Support policy
 
-All security bugs should be reported to the Microsoft Security Response Center (MSRC) at [https://aka.ms/opensource/security/create-report].
-Also see the [security policy in the NuGet.Client repo](https://github.com/NuGet/NuGet.Client/blob/dev/SECURITY.md).
+We are fully committed to supporting the most recent versions of the NuGet Client SDK packages.
+This means you can rely on us for bug fixes, updates, and enhancements exclusive to the version currently under development.
+
+NuGet Client SDK packages follow the [Microsoft Modern Lifecycle Policy](https://aka.ms/lifecycle).
```

Comment:
I disagree that the NuGet Client SDK packages follow the modern lifecycle policy. Our VS, .NET SDK, and nuget.exe experiences do, but I don't think the client SDK packages do.

Specifically, when I follow the link and look at the modern policy, it says "Microsoft's policy is to provide a minimum 30 days' notification when customers are required to take action in order to avoid significant degradation to the normal use of the product or service."

Since we're open source, does public commits on github constitute notifications with regards to the NuGet Client SDK?  Or would we have to put announcements in the annoncements repo and blog posts as well?

Also, what does "significant degradation" mean in the context of a package? My best guess is it means breaking changes to public APIs, since calling APIs is the only "normal use" of a package proving APIs.

But we will not provide 30 days notice of changing when we have to break an API for a security fix. Following the policy would also prevent us from making late changes (within 30 days of shipping) that require breaking changes to a public API for a non-security bug fix or new feature.

So, I think you should delete this line, to the lifecycle policy on the client SDK page.

Reply:
I can delete it here, but that was already stated in the Blog post which I copied it from. @JonDouglas & @kartheekp-ms do we need to follow-up with that blog post as well?

Reply:
Thanks for bringing this up Andy. I agree with your feedback. I guess we should also update the blog post based on discussion on this thread. 

Given that NuGet SDK docs has already support policy section in the docs, how about just adding `We will release patched versions of NuGet Client SDK packages only when critical security fixes are required for a long-term support (LTS) version of Visual Studio or .NET SDK.` as per the blog post, https://devblogs.microsoft.com/nuget/announcing-nuget-exe-and-nuget-client-sdk-packages-support-policy-keeping-you-informed-and-secure/#security-patch-releases.

I think we should also mention about NuGet SDK package deprecation in the support policy as mentioned in the blog post https://devblogs.microsoft.com/nuget/announcing-nuget-exe-and-nuget-client-sdk-packages-support-policy-keeping-you-informed-and-secure/#package-deprecation

Reply:
> We will release patched versions of NuGet Client SDK packages only when critical security fixes are required for a long-term support (LTS) version of Visual Studio or .NET SDK.

We often publish packages any time we service an already GA version of NuGet, even if it doesn't have security fixes. For example, the P0 bug on VS17.9 is going to create new packages, which I guess we might publish packages for, but it's not a security fix.

I don't know what good wording is, but the end result is that our packages are a byproduct of the work that we do support. We'll fix security bug, sure. But anything else, we're not going to prioritize. If a customer can show us a VS, .NET SDK or nuget.exe scenario that hits the bug, it has a chance of being worked on. But if it only reproduces by writing a custom app that uses the NuGet SDK packages directly, then it's not going to be scheduled in a sprint.

Reply:
Donnie added `Patched versions of NuGet Client SDK will be released exclusively when critical bugs or security fixes are required for a long-term support (LTS) version of Visual Studio or .NET SDK.` statement in the latest commit. I think it addresses this comment. Thanks for resolving this comment, Donnie.